single-pool/program/src/instruction.rs

```diff
@@ -43,6 +43,19 @@ pub enum SinglePoolInstruction {
     ///  12. `[]` Stake program
     InitializePool,
 
+    ///   Restake the pool stake account if its validator is force-deactivated and later
+    ///   recreated with the same vote account.
```

Comment:
feel free to suggest different wording for this, im not familiar enough with the mechanism that causes this to happen to craft proper verbiage